tests/routers/test_remote.py

```
@@ -584,114 +782,97 @@ def test_pull_file_success(
-@patch("app.routers.api.remote.db.query") # Mock db.query to avoid actual DB calls for MonitoredPath
-def test_exposed_paths(mock_db_query, authenticated_client: TestClient, monitored_path_factory):
+def test_exposed_paths(authenticated_client: TestClient, monitored_path_factory, tmp_path):
     """Test the /exposed-paths endpoint (inter-instance)."""
-    # Create a mock for the MonitoredPath objects that db.query.filter.all() would return
-    mock_path1 = MagicMock(id=1, name="Path A", enabled=True)
-    mock_path2 = MagicMock(id=2, name="Path B", enabled=True)
-    
-    # Configure the mock db.query to return these mock paths
-    mock_db_query.return_value.filter.return_value.all.return_value = [mock_path1, mock_path2]
+    # Create valid paths in DB
+    path1 = monitored_path_factory("Path A", str(tmp_path / "path_a"))
```

Ruff (0.15.1)

[error] Local variable `path1` is assigned to but never used

Remove assignment to unused variable `path1`

(F841)